README.md

```diff
@@ -25,7 +25,7 @@
 # 📖 Blog App <a name="about-project"></a>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215249978-73976410-9756-4f50-a6fd-e28c8a23b7d0.png)

**[Optional]**

- [ ] It would be nice if you remove this **NOTE** because it was just for you on how to make a license file :+1: After creating a license file you are supposed to remove this **NOTE** because it is unncessary and not make your readme clean :thinking: